scripts/kernel_monitor/kernel_monitor.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Linux Kernel LTS monitor script.
+
+Usage: kernel_monitor.py [--server=HOSTNAME] [--log-file=PATH]
+                         [--sender=ADDRESS] [--recipient=ADDRESS]
+                         [--recipient=ADDRESS] [--recipient=ADDRESS]
+                         [--recipient=ADDRESS] [--recipient=ADDRESS]
+       kernel_monitor.py (-h | --help)
+
+
+
+Options:
+  -h --help              Show this screen.
+  --log-file=PATH        Path of a file to dump logs [default: ./kernel_monitor.log]
+  --server=HOSTNAME      SMTP server ip or hostname [default: 127.0.0.1]
+  --sender=ADDRESS       e-mail sender [default: no-reply@linuxlts.mon]
+  --recipient=ADDRESS    e-mail recipients [default: root@localhost]
+
+"""
+
+import os
+import logging
+import re
+import fileinput
+import feedparser
+import smtplib
+import semver
+import json
+import urllib3
+
+from docopt import docopt
+from email.mime.text import MIMEText
+from tempfile import mkstemp
+from shutil import move
+from os import fdopen, remove
+
+# Parent directory of this script
+script_path = os.path.dirname(os.path.realpath(__file__))
+versions_file=script_path + "/../../versions.txt"
+
+# The folowing xml file is used to fetch the latest Linux kernel info
+kernel_xml_url = "https://www.kernel.org/feeds/kdist.xml"
+
+# distro_certs holds the path of the certificates bundle file provided
+# by the distro of your choice. This is needed to retrieve the xml file.
+# Fedora => /etc/ssl/certs/ca-bundle.crt
+# Ubuntu => /etc/ssl/certs/ca-certificates.crt
+distro_certs = "/etc/ssl/certs/ca-bundle.crt"
```

Comment:
- This won't work for Clear Linux.
- I think we need a CLI option to specify this path (or have it auto-detected based on distro?)